functions.js

```diff
@@ -0,0 +1,181 @@
+/*
+
+filedrag.js - HTML5 File Drag & Drop demonstration
+Featured on SitePoint.com
+Developed by Craig Buckler (@craigbuckler) of OptimalWorks.net
+
+*/
+(function () {
+	// - - - - - - - - - - - - - - - - - - - - - //
+	// HOST AND PORT
+	//HOST = 'neuralmechanics.ai/abscbn-api';
+
+	HOST = 'http://127.0.0.1:5000';
+
+	// URL Calls to flask server
+	URL_ZIP_FILE = HOST + "/zip_file"
+	URL_TRAIN = HOST + "/train"
+	URL_ANALYZE = HOST + "/analyze"
+	// - - - - - - - - - - - - - - - - - - - - - //
+
+	var output_data = [];
+	var foamtree;
+
+	// getElementById
+	function $id(id) {
+		return document.getElementById(id);
+	}
+
```

Comment:
@mismabs here is the defintion of the $id, we use this function for calling the DOM in the HTML file and add functionality into it.